streams/src/main/java/org/apache/kafka/streams/kstream/UnlimitedWindows.java

```diff
@@ -32,9 +32,8 @@
     /** The start timestamp of the window. */
     public final long start;
 
-    private UnlimitedWindows(String name, long start) {
-        super(name);
-
+    private UnlimitedWindows(long start) {
+        super();
```

Comment:
I'll leave this as is since it seems to be the preferred way in streams for now.